ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Expected Behavior
+
+## Actual Behavior
+
+## Steps to Reproduce Behavior
+
+## Issue Severity Classification - Select from:
+### (1) Severe   - Fully impedes usage of `ProjectTemplate`. This condition requires an immediate solution.  `ProjectTemplate` is unable to operate or caused other critical software to fail and there is no acceptable way to work around the problem.
```

Comment:
Could you replace `### (1)` with simply `1.` (and so on for the other numbers). The text is now marked as headers, which looks pretty bad on the eye.

Reply:
Yep, will do. PR to be sent shortly.